Makefile

```diff
@@ -33,6 +33,10 @@ EXTRA_CFLAGS += -I$(src)/hal/phydm
 
 EXTRA_LDFLAGS += --strip-debug
 
+ifneq ($(KERNELRELEASE),)
```

Comment:
What's the difference with using this and `KVER ?= $(shell uname -r)`?
(This is just ignorance on my part)